odt-opt-in/ODT Opt-in JSON Contract.md

```diff
@@ -173,16 +173,25 @@ UX returns updated state of each share. Share IDs listed are to be enrolled into
 
 ## Error Codes
 
-| Error Code | Logging Error Message                                     |
-| ---------- | --------------------------------------------------------- |
-| 501        | Config file read error: _[read error]_                    |
-| 502        | Config file validation error: _[validation error detail]_ |
-| 503        | No eligible shares                                        |
-| 504        | Ineligible account type                                   |
-| 505        | Account warning found                                     |
-| 506        | Error attempting to update share tracking record          |
-| 507        | Error updating source code and auth & fee fields          |
-| 508        | Error updating share overdraft tolerance amount           |
+*See the Modifier section for additional details.
+ The Modifier is appended to the main Logging Error Message.
+
+| Error Code | Logging Error Message                                                                 | Modifier                                                                             | Additional Notes As Needed                    |
+| ---------- | ------------------------------------------------------------------------------------- |------------------------------------------------------------------------------------- |-----------------------------------------------|
+| 500        | Program running in memo mode |||
```

Comment:
That is true. It makes more sense to make the contract that way. I've made the JSon markdown contract so it only has 500 and 503 error codes. It will be in the next commit.

Reply:
@jordan-keenan Sounds good Jordan.